src/alg_utils.jl

```diff
@@ -487,7 +487,7 @@ function default_controller(alg, cache, qoldinit, _beta1=nothing, _beta2=nothing
   if ispredictive(alg)
     return PredictiveController()
   elseif isstandard(alg)
-    return IController()
+    return StandardIController()
```

Comment:
isn't it a P controller? @YingboMa

Reply:
Citing from Hairer-Nørsett-Wanner 2, page 28:

> We saw that it was an I-controler ... and a control-man knows ... that PI is always better than I  (K. Gustafsson, June 1990)

If we order the letters of `PID` in the order of their terms, it should be `IPD` (as far as I know).

Reply:
Interesting. But it's using the proportion of the error to the tolerance to do the next step?

Reply:
Yes, that's an I-controller.